egs/chime6/asr1/local/run_beamformit.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
```

Comment:
We may not need this?

Reply:
Yes, this file is redundant. We removed it and some other unused scripts.